assets/prototype/domain/shared/predicates/tickets/filters/archivedOnly.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Internal dependencies
+ */
+import isArchived from '../isArchived';
+
+const archivedOnly = (ticketEntities: any[]) => {
+	return ticketEntities.filter((ticketEntity) => isArchived(ticketEntity));
```

Comment:
```suggestion
	return tickets.filter((ticket) => isArchived(ticket));
```

would need to adjust the function declaration too